native/packages/react-native-bpk-component-button/src/common-types.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import { ViewPropTypes } from 'react-native';
+import PropTypes from 'prop-types';
+import { type Theme } from 'react-native-bpk-theming';
+
+import { themePropType } from './utils';
+
+export const BUTTON_TYPES = {
+  primary: 'primary',
+  secondary: 'secondary',
+  destructive: 'destructive',
+  featured: 'featured',
+};
+
+export type ButtonTypeValue = $Keys<typeof BUTTON_TYPES>;
```

Comment:
I'd just call this `ButtonType`. The word values is just confusing here

Reply:
I stole that from your code 😂  but you're right, will change.